### DISCUSSION TITLE
https://blog.3place.cn/3place/sszs/SSZS007.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#108] [General]
# https://blog.3place.cn/3place/sszs/SSZS007.html

https://blog.3place.cn/3place/sszs/SSZS007.html

**Comment by picklecai** [comment]
可惜他们后来都不是专习英国文学，不然，我可真“抖”了！

抖是啥意思？

**Comment by picklecai** [comment]
青年人往往因感情的冲动，理智便被压抑了。中国学校的风潮，大多数是由于这种原因。学校中少一分风潮，便多一分成就。盼望你注意矫正这种流弊。

这样的老师，这样的语重心长！！神往！！
